### ISSUE TITLE
new galil driver: soft limits preventing a home

### ISSUE THREAD

**Issue by rerpha** [ISISComputingGroup/IBEX#8550]
As a user i'd like to be able to home without caring about soft limits. 
currently if i have an axis with soft limits on, then try and home, it seems to be able to go to the forward limit switch then struggles to come back and doesn't finish homing, printing out to the logs that the home was aborted. 

### acceptance criteria
- [ ] homing does not care about soft limits and just carries on homing

**Comment by FreddieAkeroyd** [comment]
May be  best to wait for #8701 to complete

**Comment by Tom-Willemsen** [comment]
comment to make project board checks happier

**Comment by Tom-Willemsen** [comment]
comment to make project board checks happier

**Comment by GRyall** [comment]
Is this now waiting on #8752 @FreddieAkeroyd?

**Comment by FreddieAkeroyd** [comment]
yes, it could be checked as part of this. soft limits are disabled during a home that is done in galil code, so i can only guess this is part of the post home sequence. I can see something in the jogAfterHome section that migth cause an issue, was this axis set to "jogAfterHome" ?

**Comment by KathrynBaker** [comment]
Discussed at standup as waiting on #8752 this is still impeded

**Comment by GRyall** [comment]
Continues to wait for #8752

**Comment by Tom-Willemsen** [comment]
> > > > > Continues to wait for [#8752](https://github.com/ISISComputingGroup/IBEX/issues/8752)

**Comment by FreddieAkeroyd** [comment]
comment

**Comment by GRyall** [comment]
> comment

Still waiting for #8752?

**Comment by Tom-Willemsen** [comment]
> > comment
> 
> Still waiting for [#8752](https://github.com/ISISComputingGroup/IBEX/issues/8752)?

**Comment by KathrynBaker** [comment]
And #8752 is still being waited for (this status might actually be due a birthday cake soon)

**Comment by FreddieAkeroyd** [comment]
Believe fixed and confirmed as part of CHIPIR jaws test,  i think main issue was a timeout and then it thinking that it was a soft limit issue, but soft limits are disabled in galil code. All work has moved on and is consolidated in #8979
